generators/app/index.js

```diff
@@ -367,6 +367,15 @@ module.exports = class extends BaseBlueprintGenerator {
                         debug: this.isDebugEnabled,
                     });
                 }
+                if (!this.configOptions.skipComposeE2E) {
```

Comment:
@mshima could you please confirme us (or not) that this is how we should now you modify the compose part. thanks

Reply:
Seems fine to me, but why not call it 'cypress' instead of 'e2e'.
And move this compose part to the client generator something like 
```
if (this.configOptions.skipComposeCypress || !this.clientTestFrameworkCypress) {
   return;
}
this.composeWith(require.resolve('../cypress'), {
.
.
.
}
```

Reply:
I did not init the Cypress feature, this is why I don't have the answer but I suppose is to avoid what we have done in the past with protractor which is highly linked to each client framework.

Maybe e2e subGen could host other than cypress in the futur

Reply:
So if there is no objections you should rename to cypress.
And move to the client generator.

Reply:
it seems legit. I'll do it

Reply:
Done, thanks Marcelo it's clearly better!